src/styles/style.scss

```diff
@@ -399,3 +373,19 @@ input[type='radio'] {
   left: 80%;
   position: relative;
 }
+
+// common consent styles
+.icon-wrapper {
+  margin-bottom: 20px;
+}
+
+.bottom-space {
+  margin-bottom: 10px !important;
```

Comment:
It seems like material-ui styles override these; i had to use !important to apply the rule. have you seen this behaviour elsewhere? @jwilliam706